.github/workflows/scripts/update-versions.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+
+CHARTJSON=$SCRIPTPATH/../../tests/charts.json
```

Comment:
```suggestion
CHARTJSON="${SCRIPTPATH}/../tests/charts.json"
```

resolves shellcheck warning + update when scripts folder is moved one folder up.

Reply:
Can you make a pr that does shellcheck stuff? Seems like it could be useful.